content/topics/intro-to-css-architecture/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Intro to CSS architecture
+ready: true
+---
+
+## Pre-workshop readings
```

Comment:
To programmers, loose coupling is important. 

Just list the resources and tell people to pay attention. They should read this stuff the same way whether there is a workshop or not. They should pay attention to the main concepts regardless of workshop scheduling.